### ISSUE TITLE
Support Unsplash collections

### ISSUE THREAD

**Issue by amine250** [AlfredoSequeida/venus#14]
Hello,
First of all, thanks for this amazing tool.

I wonder if it's possible to support Unsplash collections since the wallpapers curated there are prettier and of a better quality.
I could see an API available for it: https://unsplash.com/documentation#get-a-collections-photos, I haven't looked through the code, but I guess it would be just a matter of an addition to the configuration file.

Cheers

**Comment by AlfredoSequeida** [comment]
Yeah, this should be easy to accomplish. When I have some time available I'll implement that.

**Comment by AlfredoSequeida** [comment]
Oh I see you already wrote the code for it, I'll look over that as soon as I can. Thanks for contributing!
